### ISSUE TITLE
Sca generally for items

### ISSUE THREAD

**Issue by Twski** [Maxx53/SCMBot#387]
I would like to know if it's possible to edit this bot in such a way that it could scan not for a specific item, but any item that shows in a filtered market search.

For example: If I go to the Market, click in Advanced Search, select CSGO, then StatTrak, it searches for any CSGO weapon that is StatTrak, giving me this link:
http://steamcommunity.com/market/search?q=&category_730_ItemSet%5B%5D=any&category_730_TournamentTeam%5B%5D=any&category_730_Weapon%5B%5D=any&category_730_Quality%5B%5D=tag_strange&appid=730#p1_price_asc
And I know that none of them sells for under X dollars. So I set the bot to scan that page and if it finds anything there for under X dollars, it buys them.

So basically it's not scanning lots of items, it's scanning only one. But instead of an item, it's a specific search.

**Comment by Maxx53** [comment]
Everything is possible. But you need basic programming knowledge.

**Comment by Twski** [comment]
Guess you're not doing it for me :B 
Well, I know what an if is! That's enough... right?

Jokes aside, all I ever made was some basic C and Arduino programs. It would take too long for me to learn how does your (or any Windows application) really work, like what are all those .cs and what each of them do... For a begginer, reverse programming would be harder than programming something you started, I think.
Anyways, I can't afford the time to learn any of these stuff atm. But I would like to thank your effort for doing this nice bot and making it free and open source!
